$NetBSD$

Remove the boost directory reference.

Not required in pkgsrc, since boost libraries are directly installed
in ${PREFIX}/include

--- cmake/CMakeLists.txt.orig	2019-06-09 15:46:42.350731953 +0000
+++ cmake/CMakeLists.txt
@@ -25,7 +25,7 @@ else()
     include(${DCMAKE})
 endif(TARGET)
 
-set(BOOST_INCLUDEDIR ${lib_base}/boost_1_54_0)
+set(BOOST_INCLUDEDIR ${lib_base})
 find_package(Boost REQUIRED)
 
 # Include
